pkg/slices/slices.go

```diff
@@ -0,0 +1,56 @@
+package slices
+
+import (
+	"golang.org/x/exp/constraints"
+	"golang.org/x/exp/slices"
+)
+
+// Map applies a function to each item in a slice and returns a new slice with the results.
+func Map[T, V any](items []T, f func(T) V) []V {
+	result := make([]V, len(items))
+
+	for i, item := range items {
+		result[i] = f(item)
+	}
+
+	return result
+}
+
+// ElementsMatch returns true if two slices have the same elements in the same order.
+// Note that this function SORTS the slices before comparing them.
+func ElementsMatch[T constraints.Ordered](a, b []T) bool {
+	if len(a) != len(b) {
+		return false
+	}
+
+	slices.Sort(a)
+	slices.Sort(b)
+
+	for i, v := range a {
+		if v != b[i] {
+			return false
+		}
+	}
+
+	return true
+}
```

Comment:
We can have a simple unit test for this, using sample values 

same of the other function in this package